iosApp/README.md

```diff
@@ -8,6 +8,10 @@ This is an example testbed app that consumes the Messenger IO SDK framework from
 
 This project depends on a `deployment.properties` file in the parent KMM project directory. Before building this project that file must exist. There is a template file in that same directory called `deployment.properties.example` that can be used to define the needed properties.
 
+## Okta Properties
+
+Providing an `okta.properties` file at the project's root directory is required to enable the Authenticated Message behavior. However, it is not mandatory to use this feature, and the file is only necessary if you intend to utilize this functionality. An example file named okta.properties.example can be found in the same folder to help you get started.
```

Comment:
I'd recommend specifying that the file is located in the parent KMM project root (similarly stated for deployment.properties above) since it could be confusing as to whether the project root is talking about the encompassing KMM project root or the iosApp project root.

Reply:
Good point